spec/clang_spec.lua

```diff
@@ -178,4 +177,237 @@ describe("cursor kind", function()
                 end)
                 assert.are.same(expected, children)     
         end)
+end)
+
+function get_last_child(cursor)
+        local last_child
+        cursor:visitChildren(function(cursor, parent)
+                last_child = cursor
+                return "continue"
+        end)
+        return last_child
+end
+
+describe("type object creation", function() 
```

Comment:
We could be more straight-forward given that each describe covers a different function. This make easier to map which tests covers which functions.
```suggestion
describe("cursor:getType", function() 
```
Please fix all